src/backend/templates/event.hbs

```diff
@@ -32,10 +32,11 @@
     </head>
 
     <body onload="datafetcher()">
+      <a href="#content" class="skip">Skip to content</a>
     {{> scroll}}
     <a id="top"></a>
     {{> navbar}}
-    <div class="carousel slide" style="margin-bottom: 0;">
+    <div class="carousel slide" style="margin-bottom: 0;" id="content">
```

Comment:
@aayusharora thanks for the review should I send a new PR again with these corrections.